content/applications/productivity/discuss/chatter.rst

```diff
@@ -0,0 +1,407 @@
+are only accessible to internal users, and are available on any records that feature a chatter
+thread.
+
+To log an internal note, first navigate to a record. For example, to open a *CRM* opportunity,
+navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban card of an
+opportunity to open it. Then, at the top-right above the chatter composer, click :guilabel:`Log
+note`.
+
+Enter the note in the chatter composer. To tag an internal user, type `@`, then select a name from
+the drop-down menu. Depending on their notification settings, the user is notified by email or
+through Odoo.
+
+.. tip::
+   Outside contacts can also be tagged in an internal log note. The contact then receives an email
+   with the contents of the note they were tagged in, including any attachments added directly to
+   the note. If they respond to the email, their response is logged in the chatter, and they are
+   added to the record as a follower.
+
+   Outside contacts are **not** able to log in to view the entire chatter thread, and are only
+   notified of specific updates, based on their :ref:`follower subscription settings
+   <discuss/edit-subscription>`.
+
+.. _discuss/send-messages:
+
+Send messages
+=============
+
+Chatter *composers* can send messages to outside contacts, without having to leave the database, or
+open a different application. This makes it easy to communicate with potential customers in the
+*Sales* and *CRM* applications, or vendors in the *Purchase* app.
+
+To send a message, first navigate to a record. For example, to send a message from a *CRM*
+opportunity, navigate to :menuselection:`CRM app --> Sales --> My Pipeline` and click on the Kanban
+card of an opportunity to open it. Then, at the top-right above the chatter composer, click
+:guilabel:`Send message`.
+
+.. tip::
+   Press :command:`Ctrl + Enter` to send a message, instead of using the :guilabel:`Send` button.
+
+If any :ref:`followers <discuss/add-followers>` have been added to the record, they are added as
+recipients of the message.
+
+.. warning::
+   :ref:`Followers <discuss/add-followers>` of a record are added as recipients of a message
+   automatically. If a follower should **not** receive a message, they must be removed as a follower
+   before the message is sent.
+
+.. image:: chatter/send-message-followers.png
+   :align: center
+   :alt: A chatter composer preparing to send a message to the followers of a CRM opportunity and
+         the customer listed on the opportunity record.
+
+Expand full composer
+--------------------
+
+The chatter composer can be expanded to a larger pop-up window, allowing for additional
+customizations.
+
+To open the full composer, click the :guilabel:`expand` icon in the bottom-right of the composer
+window.
+
+.. figure:: chatter/chatter-expand-icon.png
+   :align: center
+   :alt: A chatter composer with emphasis on the expand icon.
```

Comment:
Might be helpful to actually show the full composer and include the expand icon as an emoji instead.

Reply:
Option to add on addition email recipients in expand composer.